src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -28,7 +28,7 @@ import { IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
 import { IWorkbenchThemeService } from 'vs/workbench/services/themes/common/workbenchThemeService';
 import { INotebookModel } from 'sql/workbench/services/notebook/browser/models/modelInterfaces';
 import { INotebookFindModel } from 'sql/workbench/contrib/notebook/browser/models/notebookFindModel';
-import { IDisposable, DisposableStore, dispose } from 'vs/base/common/lifecycle';
```

Comment:
All disposing is consolidated view DisposableStore now so that direct call to dispose is no longer needed.